package.json

```diff
@@ -417,17 +421,17 @@
                 },
                 {
                     "command": "appService.CreateSlot",
-                    "when": "view == azureResourceGroups && viewItem =~ /appService.*deploymentSlots/",
+                    "when": "view == azureResourceGroups && viewItem == deploymentSlots",
```

Comment:
Is it supposed to be `== deploymentSlots`?  Won't it have the prefix?

Reply:
It doesn't need a prefix since in functions the context value is `azFuncSlots`. Deployment slots tree items aren't shared between extensions.

Reply:
I'm surprised that we didn't share that logic between packages.  Weird decision on our parts..